packages/icons/package.json

```diff
@@ -37,7 +39,11 @@
     "@radix-ui/react-icons": "^1.1.0",
     "@webstudio-is/css-vars": "*"
   },
-  "main": "lib/index.js",
+  "exports": {
```

Comment:
This creates problems and those packages, with the current output, are not actually loadable at all - with `exports` you need explicit extension.

```ts
// test.mjs
import * as Icons from "@webstudio-is/icons";

console.log(Icons);
```

The snippet above fails with:
> Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/webstudio-designer/packages/icons/lib/legacy' imported from /webstudio-designer/packages/icons/lib/index.js

And the same thing happens for:
```js
// test.cjs
console.log(require("@webstudio-is/icons"));

```

Reply:
Oops, I guess designer storybook with webpack 5 just caught this issue